I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 35, in script
    python:
  File "game/script.rpy", line 39, in <module>
    if not povname:
NameError: name 'povname' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 35, in script
    python:
  File "C:\Users\bitin\Downloads\renpy-7.4.11-sdk/renpy/ast.py", line 928, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\bitin\Downloads\renpy-7.4.11-sdk/renpy/python.py", line 2245, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 39, in <module>
    if not povname:
NameError: name 'povname' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
dating sim 1.0
Sat Mar 26 15:52:51 2022
